main/bind/APKBUILD

```diff
@@ -174,30 +182,58 @@ dev() {
 	mv "$pkgdir"/usr/bin/isc-config.sh "$subpkgdir"/usr/bin/
 }
 
-_py3() {
-	pkgdesc="A module allowing rndc commands to be sent from Python programs"
-	depends="$_py3deps"
+plugins() {
+	pkgdesc="The ISC DNS server plugins"
+	depends="$depends_plugins"
+
+	mkdir -p "$subpkgdir"/usr/lib
+	mv "$pkgdir"/usr/lib/bind "$subpkgdir"/usr/lib/
+}
+
+py3_isc() {
+	pkgdesc="Python functions to support BIND utilities"
+	depends="$depends_py3_isc"
+	provides="py3-isc=${_isc_ver}-r${_isc_rel} py3-$pkgname=$pkgver-r$pkgrel"
```

Comment:
why do you provide different version of py3-isc? This will create trouble and I cannot see the value it brings.

Reply:
> why do you provide different version of py3-isc? This will create trouble and I cannot see the value it brings.

It has its own version. Why should it be given the version of `bind`? Just because it is bundled?

Ignoring the version of the python `isc` module is more likely to create trouble in my opinion.